plugins/ccgx/fu-ccgx-hpi-device.c

```diff
@@ -308,25 +264,30 @@ fu_ccgx_hpi_device_i2c_write (FuCcgxHpiDevice *self,
 			      CyI2CDataConfigBits cfg_bits,
 			      GError **error)
 {
-	guint8 i2c_status = 0x0;
-	guint8 slave_address;
-	g_autoptr(GError) error_local = NULL;
+	guint8 i2c_status[CY_I2C_EVENT_NOTIFICATION_LEN] = {0};
+	guint8 slave_address = 0;
 
+	i2c_status[0] = 0;
 	if (!fu_ccgx_hpi_device_get_i2c_status (self,
 						CY_I2C_MODE_WRITE,
-						&i2c_status,
+						i2c_status,
 						error)) {
 		g_prefix_error (error, "i2c get status error: ");
 		return FALSE;
 	}
-	if (i2c_status & CY_I2C_ERROR_BIT) {
+	if (i2c_status[0] & CY_I2C_ERROR_BIT) {
+		if (!fu_ccgx_hpi_device_reset_i2c (self, CY_I2C_MODE_WRITE, error)) {
+			g_prefix_error (error, "i2c reset error: ");
+			return FALSE;
+		}
 		g_set_error (error,
-			     FWUPD_ERROR,
-			     FWUPD_ERROR_INTERNAL,
-			     "i2c status error in i2c write: 0x%x",
-			     (guint8) i2c_status);
+				FWUPD_ERROR,
+				FWUPD_ERROR_INTERNAL,
+				"i2c status error in i2c write: 0x%x",
+				(guint8) i2c_status[0]);
```

Comment:
I don't think you needed to change all this whitespace, just add the [0]

Reply:
I will remove white space and make it aligned.

Reply:
I fixed it.